### ISSUE TITLE
PXPay - getMessage not returning errors.

### ISSUE THREAD

**Issue by Acen** [thephpleague/omnipay-paymentexpress#30]
getMessage is seemingly being ignored and not returning any errors from the payment processor.

**Comment by gdhnz** [comment]
I've found there's 2 types of errors from PxPay.

The first type is when there's an issue creating the transaction.  In this case, the error code and error message are returned. For example, the username or password are incorrect.

The 2nd type of error is when someone tries a credit card and it's declined for whatever reason.  In this instance, the processor only returns the status code and it's up to the developer to display a message.  The provider doesn't return these error codes in the xml.

I've contacted the payment processor and they've provided a list of the 2nd type of error codes:
```
array(
      '00' => 'Transaction is approved and successful.',
      '01' => 'The transaction was not approved. Please contact the issuer of the credit card.',
      '04' => 'Refer To Card Issuer.',
      '05' => 'Card DECLINED on banks end.',
      '14' => 'Invalid Card Number.',
      '51' => 'Insufficient Funds.',
      '54' => 'Expired card or expiry date incorrect.',
      '58' => 'Function Not Permitted To Terminal.',
      '65' => 'Acceptor Contact Acquirer, Security.',
      '76' => 'Possible expiry date or pin problem. Not enough money on purchase.',
      'AK' => 'No BIN Range match found for card in port list / Auth not enabled on port.',
      'AQ' => 'Amex is not enabled on the port.',
      'AU' => 'Discover not Accepted.',
      'LR' => 'Local Risk Management Declined.',
      'NU' => 'Local Fail (Reject Card Number).',
      'QK' => 'Invalid Card Number invalid length. If live, it could be invalid expiry date.',
      'RC' => '3D Secure error/Fusion Cancellation/FEP Declined.',
      'U9' => 'Uplink Timeout.',
      'Y7' => 'Invalid Card Number (Incorrect Length).',
      'YT' => 'Invalid CVC2.'
    )
```

**Comment by eileenmcnaughton** [comment]
This improves the error reporting where there is no URI set https://github.com/thephpleague/omnipay-paymentexpress/pull/32/files

**Comment by delatbabel** [comment]
Merged #32
